types/index.d.ts

```diff
@@ -33,16 +33,17 @@ export interface RouteProps {
 }
 export const Route: FunctionComponent<RouteProps>;
 
-export interface LinkProps extends AnchorHTMLAttributes<HTMLAnchorElement> {
-  to?: Path;
-  href?: Path;
-}
+export type NavigationalProps =
```

Comment:
```typescript
export type NavigationalProps = { to: Path } | { href: Path}
```

Reply:
It needs to be tested, I'm sure that it works, but not 100%

Reply:
Well, this is tested via the `Link` component tests. See comments below.